ext/gd/tests/webp_basic.phpt

```diff
@@ -14,6 +14,8 @@ if (!function_exists('imagewebp') || !function_exists('imagecreatefromwebp'))
 <?php
 require_once __DIR__ . '/similarity.inc';
 
+define("WEBP_LOSSLESS", 101);
```

Comment:
Should probably define this as a constant in the extension (see REGISTER_LONG_CONSTANT in gd.c), so users don't need to know a magic number?

Reply:
Yes. Thanks for this!